src/components/CreateList.vue

```diff
@@ -0,0 +1,77 @@
+<template>
+  <div @click="closeForm" class="fullscreen">
+    <form v-on:submit.prevent="">
+      <input
+        v-focus
+        type="text"
+        v-model="listTitle"
+        placeholder="Ente the list name"
+        @keyup.enter="onClick"
+      >
+      <button @click="onClick">Create List</button>
+    </form>
+  </div>
+</template>
+
+<script>
+export default {
+  name: 'CreateTask',
+  data: () => ({
+    listTitle: '',
+  }),
+  methods: {
+    onClick() {
+      this.$emit('getListTitle', this.listTitle);
+      this.listTitle = '';
+    },
+    closeForm(e) {
+      if (e.target.className === 'fullscreen') {
```

Comment:
Much better to compare event.target with event.currentTarget instead of className check. But using Vue you can easily add self event modifier to event listener (https://vuejs.org/v2/guide/events.html#Event-Modifiers)